container-guide/concepts/containers-intro.xml

```diff
@@ -160,8 +160,8 @@
     </orderedlist>
     <para>
       To run a container, you need an image. An image includes all dependencies
-      needed to run the application. For example, the &bcia;-Base image
-      contains the &slea; distribution with a minimal package selection.
+      needed to run the application. For example, the &bcia;-Base image contains
+      the &slea; distribution with a minimal package selection.
```

Comment:
The output _For example, the **SLE BCI-Base image** contains the SLE distribution with a minimal package selection._ confuses me again.